### DISCUSSION TITLE
upload your demo video onto youtube

### DISCUSSION THREAD

**Discussion by alberttwong** [slingdata-io/sling-cli#168] [Ideas]
I'm trying to add your video at https://cdn.dorik.com/5f09f3e39ac5fa001196a14b/6060fc12ae30280011f77e6e/videos/sling.demo.2023.10.720_g8abit3z.mp4#t=0.1 to a medium article but it only allows me to embedd youtube videos.  Can you upload to youtube?

**Comment by flarco** [comment]
Done: https://youtu.be/3bLPCxYRF3E
